app/Http/Controllers/LabelsController.php

```diff
@@ -14,16 +14,14 @@
 use App\Models\Supplier;
 use App\Models\User;
 use App\View\Label as LabelView;
-use Illuminate\Support\Facades\DB;
-use Illuminate\Support\Facades\Storage;
 
 class LabelsController extends Controller
 {
     /**
      * Returns the Label view with test data
      *
      * @param string $labelName
-     * @return \Illuminate\Contracts\View\View
+     * @return \Illuminate\Contracts\View\View | \Illuminate\Http\RedirectResponse
```

Comment:
This is another one where we're editing the docblock rather than specifying a return-type?

Reply:
Yes, this was an oversight from my initial pass. I believe we actually go both there, but I will delete the @return